webapp/src/container.tsx

```diff
@@ -84,10 +86,34 @@ export class SideDocs extends data.Component<ISettingsProps, {}> {
     toggleVisibility() {
         const state = this.props.parent.state;
         this.props.parent.setState({ sideDocsCollapsed: !state.sideDocsCollapsed });
+        document.getElementById("sidedocstoggle").focus();
     }
 
     componentDidUpdate() {
         this.props.parent.editor.resize();
+
+        if (!this.props.parent.state.sideDocsCollapsed) {
+            this.rootNode = ReactDOM.findDOMNode(this);
+            if (this.rootNode !== null) {
+                core.initializeFocusTabIndex(this.rootNode);
+            }
+        }
+        else {
+            this.unmountSideDocs();
+        }
+    }
+
+    componentWillUnmount() {
+        this.unmountSideDocs();
+    }
+
+    unmountSideDocs = () => {
+        (document.activeElement as HTMLElement).blur();
```

Comment:
The issue I've been seeing is due to this line of code. 
Why are we doing this? 

You should be able to repro this on Windows Browsers, have a side docs open, and then try to tab to any of the simulator buttons and click on them to cause a re-render. You'll lost focus as componentDidUpdate will call unmountSideDocs and cause the blur.
You should really not be doing a blur, what's the reason for it?